README.md

```diff
@@ -1,23 +1,69 @@
-# Blog for *ASET ALiAS*
+# _ASET ALiAS Blog_
 
+This is the repository of the official blog for _Amity Linux Assitance Sapience_
 
-This static website has been created using hugo.
+ALiAS is a platform for upcoming developers for finding exposure by meeting the people working in related industries, learning various languages and becoming a better developer.
 
-*for hugo documentation [click here](https://gohugo.io/documentation/)*
+This website has been created using hugo and is currently under construction. Please contribute as much as you can. All your inputs are highly appreciated.
 
-*to get to know this theme better [click here](https://github.com/appernetic/hugo-nederburg-theme/)*
+* * *
 
-*markdown cheat sheet [click here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code)*
+## _Using the Website_
 
----
+### _For General users:_
 
-*to run this website on your system, clone the repository*
+`Hugo Installation`
 
-    >git clone https://github.com/rohanvtk/ALiAS_Blog.git
+1) [Click Here](https://github.com/gohugoio/hugo/releases) to  download latest version of Hugo for your operating system.
+2) MacOS users can also  install Hugo using Homebrew utility and Linux users can install Hugo directly from the official directory for their distribution.  _For more detailed instructions please visit [Hugo Installation](https://gohugo.io/getting-started/installing)._
+3) Create a folder named Hugo in your C:\\ drive.
+4) Create 2 subfolders: bin and Sites.
+5) Extract the contents of the downloaded zip file in C:\\Hugo\\bin.
+6) Search and open  Advanced System settings.
+7) Click on the Environment Variables… button on the bottom.
+8) In the User variables section, find the row that starts with PATH.
+9) Double-click on PATH and Click the New… button.
+10) Type in C:\\Hugo\\bin and click ok.
 
-*to install hugo [click here](https://gohugo.io/getting-started/quick-start/) for a step-by-step guide*
+_Hugo is now installed in your sytem and is ready to use_
 
-*mount the directory on terminal and type __hugo server__*
+`Running the Website`
 
+1) Download this repository or clone it if you use git.
+2) Extract the contents of the downloaded file in C:\\Hugo\\Sites.
+3) Open your terminal and type `cd C:\Hugo\Sites\ALiAS_Blog`.
+4) Now, enter the command `hugo server`.
```

Comment:
Again, since you are mentioning installation instructions for Hugo for all 3 platforms, it would be way easier if the site testing part could be there as well. With information about all **3** platforms. 

Another suggestion would be to keep the paths as general as possible. You can replace the same by `Navigate to the folder of the downloaded repository`